src/main/kotlin/hwr/oop/Card.kt

```diff
@@ -0,0 +1,11 @@
+package hwr.oop
+
+class Card (private val cardRank: CardRank, private val cardSuit: CardSuit){
+    fun rank(): CardRank {
+        return cardRank
+    }
+
+    fun suit(): CardSuit {
+        return cardSuit
+    }
```

Comment:
It would be better to create a public getter rather then custom functions.
Example:
```suggestion
var propertyName: PropertyType
    private set
```